aicsimageio/readers/tiff_reader.py

```diff
@@ -161,7 +161,7 @@ def _get_image_data(
                 )
             )
             arr = arr.transpose(transpose_indices)
-            return arr[retrieve_indices].compute()
+            return arr[retrieve_indices].compute(scheduler="synchronous")
```

Comment:
any observable performance impact in the multithreaded case?

Reply:
oops i just noticed this is marked as draft...

Reply:
In addition to this change making multi-processing possible...

It makes multi-threaded reads a tiny bit faster!

To do multi-threaded test I did:
```python
In [2]: from distributed import Client, LocalCluster

In [3]: cluster = LocalCluster(processes=False)

In [4]: client = Client(cluster)
```

Before this change:

```python
In [6]: img = AICSImage("aicsimageio/tests/resources/3d-cell-viewer.ome.tiff")

In [7]: %timeit img.get_image_dask_data("CZYX", C=[0, 1, 2, 3]).compute().shape
1.94 s ± 78.7 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
```

After this change:
```python
In [5]: img = AICSImage("aicsimageio/tests/resources/3d-cell-viewer.ome.tiff")

In [6]: %timeit img.get_image_dask_data("CZYX", C=[0, 1, 2, 3]).compute().shape
1.65 s ± 18.7 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
```

So we gain functionality AND a tiny speed increase 🙂